jobs/src/main/scala/dcos/metronome/model/Container.scala

```diff
@@ -15,3 +15,17 @@ case class DockerSpec(
 object DockerSpec {
   val DefaultParameters = Seq.empty[Parameter]
 }
+
+case class ImageSpec(
+  id:        String,
+  kind:      String  = ImageSpec.DefaultKind,
+  forcePull: Boolean = false)
+
+object ImageSpec {
+  val DefaultKind = "docker"
```

Comment:
Nit: `DefaultKind` seems an odd name when it is `docker`. Why not `ImageSpec.Docker`?

Reply:
The way it is written is consistent with the rest of the metronome code base where defaults are defined and easily understood.  It doesn't seem necessary here as clearly line 21 would indicate that it is the default.    
the file `models/pacakge.scala` shows uses of defaults for reading from JSON.   It would be clear to change to `ImageSpec.Docker`... however it would be inconsistent with everything else in that file.